src/report/mod.rs

```diff
@@ -137,6 +137,8 @@ pub fn generate_report<DB: ReadResults>(
     crates: &[Crate],
 ) -> Fallible<TestResults> {
     let shas = db.load_all_shas(ex)?;
+    let mut crates = crates.to_vec();
+    crates.sort_unstable_by(|a, b| a.id().cmp(&b.id()));
```

Comment:
Can you add a comment clarifying why an unstable sort is used?

Reply:
well, we don't need stable sorting, and it is generally faster (according to rust doc)